x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/import_rules_route.test.ts

```diff
@@ -0,0 +1,399 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { omit } from 'lodash/fp';
+
+import { getSimpleRuleAsMultipartContent } from '../../../../../../../../test/detection_engine_api_integration/security_and_spaces/tests/utils';
```

Comment:
I think at this point it would be better if:

```ts
getSimpleRuleAsMultipartContent
```

were to be lifted and added to local testing utilities here within the `detection_engine` and then the files in:

```
test/detection_engine_api_integration/security_and_spaces/tests/utils
```

just include `getSimpleRuleAsMultipartContent` from on this side. Otherwise things get weird with unit tests importing from e2e tests which are already importing from unit tests. Probably better to keep it non-circular by just moving `getSimpleRuleAsMultipartContent` here and any others that we end up using.

Reply:
That makes sense. It did feel a little icky pulling this in that way. I went ahead and moved into the test utils file under `__mocks__`.